ndcube/utils/tests/test_utils_wcs.py

```diff
@@ -112,14 +112,15 @@ def test_get_dependent_array_axes(axis_correlation_matrix):
 
 def test_get_dependent_world_axes(axis_correlation_matrix):
```

Comment:
Can you change this `axis_correlation_matrix` fixture to just write out the matrix long form? I have no idea what this test is testing because I don't know what the input is :confused: